packages/common/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0-beta.581d31f4",
   "description": "Common utilities used across tasks",
   "keywords": [
-    "GIBS"
+    "cumulus"
```

Comment:
This is different than the keywords defined in the top-level package.json.  Not sure if that matters.

```
"keywords": [
    "GIBS",
    "CUMULUS",
    "NASA"
  ]
```